src/listfunc.c

```diff
@@ -640,8 +640,10 @@ Obj             FuncPOSITION_FIRST_COMPONENT_SORTED (
 #define SORT_COMP(v, w) CALL_2ARGS(func, v, w) == True
 /* list is not necc. sorted wrt. \< (any longer) */
 #define SORT_FILTER_CHECKS() \
-  RESET_FILT_LIST(list, FN_IS_SSORT); \
-  RESET_FILT_LIST(list, FN_IS_NSORT);
+  if(IS_PLIST(list)) {\
```

Comment:
```suggestion
  if (IS_PLIST(list)) {\
```
... or just `clang-format`

Reply:
Or don't make this change, but simply fix `RESET_FILT_LIST`, it's a trivial change?

Reply:
made change